coffee-shop-service/src/index.js

```diff
@@ -7,38 +7,16 @@ import {typeDefs} from './schema';
 import * as queries from './queries.js';
 import ApolloClient from "apollo-boost";
 import fetch from "node-fetch";
-import gql from "graphql-tag";
 
-// setup global fetcher
-global.fetch = fetch;
+
 const client = new ApolloClient({
   uri: "http://localhost:5000/graphql",
+  fetch: fetch
```

Comment:
- Please revert, I want to follow the ApolloClient convention ... that this is global (inherits the one from the browser)... if anything I can add a comment.

- This is a nit... not worth your time to refactor.